apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/nodes/VideoBlockNode/VideoBlockNode.stories.tsx

```diff
@@ -0,0 +1,192 @@
+import { MockedProvider } from '@apollo/client/testing'
+import Box from '@mui/material/Box'
+import { Meta, StoryObj } from '@storybook/react'
+import { Background, ReactFlow } from 'reactflow'
+import 'reactflow/dist/style.css'
+
+import { TreeBlock } from '@core/journeys/ui/block'
+
+import {
+  BlockFields_StepBlock as StepBlock,
+  BlockFields_VideoBlock as VideoBlock
+} from '../../../../../../../__generated__/BlockFields'
+import {
+  ThemeMode,
+  ThemeName,
+  VideoBlockSource
+} from '../../../../../../../__generated__/globalTypes'
+import { journeysAdminConfig } from '../../../../../../libs/storybook'
+
+import { VideoBlockNode } from '.'
+
+const VideoBlockNodeStory: Meta<typeof VideoBlockNode> = {
+  ...journeysAdminConfig,
```

Comment:
I don't think this is explained anywhere, but FYI, we use `journeysAdminConfig` for when we need both mobile and and desktop snapshots. When we only need a desktop snapshot, we use `simpleComponentConfig`. Since the flow looks the same on all screen sizes, we use simple to cut down on the number of snapshots.

```suggestion
  ...simpleComponentConfig,
```